packages/fuels-core/src/codec/logs.rs

```diff
@@ -53,6 +99,38 @@ impl Debug for LogFormatter {
     }
 }
 
+#[derive(Clone)]
+pub struct ErrorFormatter {
```

Comment:
Can the LogFormatter be used for formatting errors as well?

Reply:
Yes good catch. Forgot to delete this.

Reply:
60694aa